From 420237601d16a15253eb513f45c15acda21a2d8b Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Thu, 14 Jan 2021 15:20:29 +0800
Subject: [PATCH] Revert "ARM: rockchip_defconfig: enable HZ_1000"

This reverts commit 52d7b98f85edf9162aa043ef587ca28f857c1181.

There is no need to keep HZ=1000 after commit 3be6453a90e4
("kthread: do not preempt current task if it is going to call schedule()").

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I42632c8dab553762574249e7fc219a50a9cf5323
---
 arch/arm/configs/rockchip_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 5d1c505bb2b5..84fc28f03d7b 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -46,7 +46,7 @@ CONFIG_ARM_ERRATA_754322=y
 CONFIG_SMP=y
 CONFIG_SCHED_MC=y
 CONFIG_ARM_PSCI=y
-CONFIG_HZ_1000=y
+CONFIG_HZ_300=y
 CONFIG_HIGHMEM=y
 CONFIG_SECCOMP=y
 CONFIG_ZBOOT_ROM_TEXT=0x0
-- 
2.35.3

